modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/ddm-templates/solutions-search-results/ddm-template.ftl

```diff
@@ -108,15 +112,15 @@
 									<#assign productPriceModels = productSpecifications?filter(item -> item.specificationKey == "developer-name") />
 
 									<#list productPriceModels as productPriceModel>
-										<div class="color-neutral-3 font-size-paragraph-small mt-1">
+										<div class="color-neutral-3 font-size-paragraph-small mt-1 developer-name">
```

Comment:
This is weird, you are assigning the color #545d69 but you have `color-neutral-3` applied to this div, which color should be used?

```suggestion
										<div class="color-neutral-3 font-size-paragraph-small mt-1 developer-name">
```